### PR TITLE
Feature/jun/seachat tutorial introduction

### PR DESCRIPTION
**Pull request by itIsJim**

### CODE REVIEW

File: content/en/seachat/getting-started/02-quick-start.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  - /seachat/seachat-intro/02-quick-start/
+---
+
+* To get a quick taste of SeaChat, you can simply log into your SeaChat account now and follow our hands-on video tutorial to start.
+* This step-by-step guide will walk you through creating your first SeaChat agent from scratch. In just under 10 minutes, you'll have your own customized AI agent set up and ready to deploy on your website.
+* For more detailed guidance, please consult the [SeaChat User Manual](/en/seachat/seachat-intro/02-how-to-create/). Should you encounter any issues or have questions, our [support team](#support) is on standby to assist you. Don't hesitate to reach out.
+
+---
+## Step 1: Create a Workspace 
+If this is your first time logging into SeaChat, you will have to create a workspace to kickstart the process. Enter the name of your workspace and let’s begin!
```
Comment (xuchen) [inline content/en/seachat/getting-started/02-quick-start.md:21]: Explain what does Workspace do?

File: content/en/seachat/getting-started/02-quick-start.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+<br/>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step2-2.png" alt="">
+<br/>
+
+<br/>
+
+> :bulb: **Note** 
+>
+>  The live agent feature will require a human agent in case the customer wishes to be transferred to one. The user will then have the option to request a human agent during the chat.
```
Comment (xuchen) [inline content/en/seachat/getting-started/02-quick-start.md:43]: this is a good note. 

Can you also clarify why we use "Agent" but not "bot"? and when we say "Agent" did we mean human agent or AI agent?

File: content/en/seachat/getting-started/01-seachat-intro.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+images: []
+aliases:
+  - /en/seachat/seachat-intro/01-introduction/
+  - /seachat/seachat-intro/01-seachat-intro/
+toc: true
+---
+
+## Welcome to SeaChat
+
+Build and launch a GPT chat and voice agent in 10 minutes. Multilingual support. Live agent transfer. Integrate with webchat, SMS, Line, CRM, Shopify, calendars, Twilio, ZenDesk, and phone calls. You can upload your knowledge base (we support more than 30 document types including pdf, csv, doc, png, jpg etc) and customize an AI agent in under 10 minutes.
```
[the remark this answers is not in the document]
Comment (xuchen) [reply content/en/seachat/getting-started/01-seachat-intro.md:18]: oh ok, Amy told me that you didn't write this.. Could you please then revise it?

File: content/en/seachat/getting-started/02-quick-start.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+<br/>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step5.png" alt="">
+<br/>
+
+<br/>
+
+Your role as a tester involves verifying if our agent produces the correct responses based on the data uploaded to its knowledge base, and determining if the responses require linkage with external data or additional information. To review a message, hover over it and provide positive feedback to your agent with a thumbs up. If the message needs further review, give it a thumbs down.
+
+<br/>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step5-2.png" alt="">
```
Comment (xuchen) [inline content/en/seachat/getting-started/02-quick-start.md:89]: when i look at the image here, I can't see the details at all... How do you fix this?

image

**Review by xuchen — CHANGES_REQUESTED** [review]
image

the png files are in general too big. Change them to .jpg to compress the size.

File: content/en/seachat/getting-started/02-quick-start.md (hunk 4 above)
Comment (amy17519) [reply content/en/seachat/getting-started/02-quick-start.md:89]: zoom in to the conversation so you can show the highlighted conversation. Like this
Screenshot 2024-03-11 at 17 44 59

File: content/en/seachat/getting-started/02-quick-start.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+weight: 202
+draft: false
+images: []
+aliases:
+  - /en/seachat/seachat-intro/02-quick-start/
+  - /seachat/seachat-intro/02-quick-start/
+---
+
+* To get a quick taste of SeaChat, you can simply log into your SeaChat account now and follow our hands-on video tutorial to start.
+* This step-by-step guide will walk you through creating your first SeaChat agent from scratch. In just under 10 minutes, you'll have your own customized AI agent set up and ready to deploy on your website.
```
Comment (amy17519) [inline content/en/seachat/getting-started/02-quick-start.md:16]: @itIsJim We want to show how to create AI agent in 10 minutes **with an actual example**. We want to use the parking example as a template to **show** people a real use case. The example files for the parking bot from Kim should be a part of this tutorial.

Just like the tutorial we all liked: https://www.youtube.com/watch?v=Edj6ijQI1hY. Let's demo an actual use case, not just listing the steps and buttons on this tutorial
Screenshot 2024-03-11 at 17 51 48

File: content/en/seachat/getting-started/01-seachat-intro.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+images: []
+aliases:
+  - /en/seachat/seachat-intro/01-introduction/
+  - /seachat/seachat-intro/01-seachat-intro/
+toc: true
+---
+
+## Welcome to SeaChat
+
+Build and launch an AI agent and voice agent that elevates your workflow in under 10 minutes. SeaChat supports more than 30 document types, including PDF, CSV, DOC, PNG, and JPG, allowing you to upload almost unlimited knowledge to train your AI chatbot tailored to your needs with a seamless transition to human agent when needed.
```
Comment (amy17519) [inline content/en/seachat/getting-started/01-seachat-intro.md:18]: Build and launch an AI agent that elevates your workflow in under 10 minutes. SeaChat supports more than 30 document types, including PDF, CSV, DOC, PNG, JPG and more, allowing you to upload almost unlimited knowledge to train your AI voice and chat agents tailored to your needs with a seamless transition to human agent when needed.

> Corrected a few grammar mistake. I think we'd eventually make this more succinct but let's keep this for SEO purpose

File: content/en/seachat/getting-started/02-quick-start.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+  - /seachat/seachat-intro/02-quick-start/
+---
+---
+* In just under 10 minutes, you'll have your own customized AI agent set up and ready to deploy on your website.
+* To get a quick taste of SeaChat, you can simply log into your SeaChat account now and follow our hands-on video tutorial to start.
+* For more detailed guidance, please consult the [SeaChat User Manual](/en/seachat/seachat-intro/02-how-to-create/). Should you encounter any issues or have questions, our [support team](#support) is on standby to assist you. Don't hesitate to reach out.
+
+---
+
+In this tutorial, we will create a SeaChat agent from scratch. This AI Agent will help us answer frequently asked questions about parking lots. We will walk you through the process of creating an AI agent, adding knowledge to its knowledge base, and testing and fine-tuning the agent to ensure it's ready for deployment.
```
Comment (amy17519) [inline content/en/seachat/getting-started/02-quick-start.md:21]: 1. This AI agent will help answer FAQs about a parking lot business. 
2. don't say "deployment". The audience are no-code users. "... to ensure it's ready to answer all incoming questions from customers!

File: content/en/seachat/getting-started/02-quick-start.md [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+lastmod: 2024-03-08 10:43:51.069 +0100
+weight: 202
+draft: false
+images: []
+aliases:
+  - /en/seachat/seachat-intro/02-quick-start/
+  - /seachat/seachat-intro/02-quick-start/
+---
+
+* To get a quick taste of SeaChat, you can simply log into your SeaChat account now and follow our hands-on video tutorial to start.
```
[the remark this answers is not in the document]
Comment (amy17519) [reply content/en/seachat/getting-started/02-quick-start.md:16]: add a section title here, the section title could be "Let's Create Your First Agent". 

Currently, when the user make the window narrower or is on his/her phone, this page currently looks like this:
Screenshot 2024-03-12 at 16 38 35



This is what Xuchen means by there's no clear separation

File: content/en/seachat/getting-started/02-quick-start.md [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+weight: 202
+draft: false
+images: []
+aliases:
+  - /en/seachat/seachat-intro/02-quick-start/
+  - /seachat/seachat-intro/02-quick-start/
+---
+---
+* In just under 10 minutes, you'll have your own customized AI agent set up and ready to deploy on your website.
+* To get a quick taste of SeaChat, you can simply log into your SeaChat account now and follow our hands-on video tutorial to start.
```
Comment (amy17519) [inline content/en/seachat/getting-started/02-quick-start.md:16]: don't direct the user to another page and watch another video. just delete 2nd bullet point

File: content/en/seachat/getting-started/02-quick-start.md [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+* For more detailed guidance, please consult the [SeaChat User Manual](/en/seachat/seachat-intro/02-how-to-create/). Should you encounter any issues or have questions, our [support team](#support) is on standby to assist you. Don't hesitate to reach out.
+
+---
+
+In this tutorial, we will create a SeaChat agent from scratch. This AI Agent will help us answer frequently asked questions about parking lots. We will walk you through the process of creating an AI agent, adding knowledge to its knowledge base, and testing and fine-tuning the agent to ensure it's ready for deployment.
+
+## Step 1: Create a Workspace 
+If this is your first time logging into SeaChat, you'll need to create a workspace to start the process. In this tutorial, we'll create a new workspace called **My Work Space**. Enter the name of your workspace and let's dive in!
```
Comment (amy17519) [inline content/en/seachat/getting-started/02-quick-start.md:24]: "If this is your first time logging into SeaChat, you'll need to create a workspace to start the process. I named my new workspace **My Work Space**. You can give your workspace a different name!"

Add a pic of workspace creation (with the workspace name "My Work Space" filled in

File: content/en/seachat/getting-started/02-quick-start.md [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+
+> **What Is a Workspace ?**
+>
+>  A workspace serves as an information hub containing your organization's data, offering an overview of your agents, members, and API keys. You have the option to either create a new workspace for yourself or join an existing one by requesting an invitation from the admin.
+
+## Step  2: Your First SeaChat Agent
+Click on **Add New Agent** to configure your new agent. If you already have a workspace and an agent, you can create a new one by clicking on the dropdown menu located in the top-right corner next to your profile icon.
+
+> :books: **Agent and Live Agent**
```
Comment (amy17519) [inline content/en/seachat/getting-started/02-quick-start.md:33]: AI Agent and Live Agent

File: content/en/seachat/getting-started/02-quick-start.md [hunk 12]
```diff
@@ -0,0 +1,143 @@
...
+>
+>You will see the term, **Agent**, used throughout this tutorial. An agent is essentially an AI assistance with a custom knowledge base, we use the term, **Agent** to separate its capabilities from a traditional chatbot. Whereas a **Live Agent** refers to a human agent who interacts with users in real-time.
+
+
+<br/>
```
Comment (amy17519) [inline content/en/seachat/getting-started/02-quick-start.md:38]: Did you figure out a way to click and enlarge the pic?

File: content/en/seachat/getting-started/02-quick-start.md [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+<img width="80%" style="border-radius: 0.4rem; transition: transform 0.5s ease;" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step2.jpg" alt="">
+<br/>
+
+<br/>
+
+Once you have picked the right name for your agent, move on to **Description** to give your agent some context of its role and functions before you choose its use case and response language. I have picked a very creative name **My Parking Agent** for this tutorial.  I will choose **FAQ** as my use case, since this agent will be used to answer frequently asked questions, and **US English** for the response language.
```
Comment (amy17519) [inline content/en/seachat/getting-started/02-quick-start.md:44]: "very creative" are you being sarcastic lol?

File: content/en/seachat/getting-started/02-quick-start.md [hunk 14]
```diff
@@ -0,0 +1,143 @@
...
+
+Feel free to try out the different ways to upload data to your knowledge base. For the purposes of this tutorial, we'll focus on utilizing the spreadsheet upload method. However, if you're interested in exploring additional data uploading techniques, we invite you to consult the [Add Knowledge]( /en/seachat/seachat-intro/03-add-knowledge/add-knowledge-intro) section within our user manual for comprehensive instructions and tips.
+<br/>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step3.jpg" alt="">
+<br/>
+
+## Step  4:  Upload Files to Knowledge Base
+Uploading a spreadsheet is as simple as clicking on the upload icon and choosing the desired spreadsheet. Once you see your uploaded files in the preview section below the drag-and-drop section, click on the **Next** button to confirm. 
+Now, ncc_Car_Parks (1) is uploaded, adn we can see each header and its corresponding data in the preview section. My Parking Agent is now ready for the next step.
```
Comment (amy17519) [inline content/en/seachat/getting-started/02-quick-start.md:68]: do you plan to upload this csv to google drive and let people download it. I think that'd be nice. If you do, make this file clickable and redirects to download

File: content/en/seachat/getting-started/02-quick-start.md [hunk 15]
```diff
@@ -0,0 +1,143 @@
...
+
+<br/>
+
+<a href="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step4.jpg" target="_blank">
```
Comment (amy17519) [inline content/en/seachat/getting-started/02-quick-start.md:72]: oh this picture can be clicked on and enlarged. make sure this behavior is applied to all the other pictures in this tutorial

File: content/en/seachat/getting-started/02-quick-start.md [hunk 16]
```diff
@@ -0,0 +1,143 @@
...
+
+<br/>
+<a href="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step4-2.jpg" target="_blank">
+<img width="80%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step4-2.jpg" alt="">
+</a>
+<br/>
+
+<br/>
+
+That’s it. Now, My Parking Agent has processed the uploaded data, and let’s set it up for a test to see if it needs further fine-tuning. Click on the **Test AI Agent** button located in the bottom-right corner. Test away and let’s fine-tune the AI agent for optimal performance!.
```
Comment (amy17519) [inline content/en/seachat/getting-started/02-quick-start.md:90]: the user might not understand why he needs to test and fine-tine the AI agent.

Try to explain the concept a bit more. 
"Before launch, let's test the AI agent's accuracy based on the uploaded knowledge. This process is called fine-tuning. Simply chat with the agent to assess its responses, and make any necessary adjustments to the knowledge base for better results.

Comment (amy17519) [inline content/en/seachat/getting-started/02-quick-start.md:90]: The language "Test away" is a bit weird here. 

"Let's try chatting with the My Parking Agent and fine-tune..."

File: content/en/seachat/getting-started/02-quick-start.md [hunk 17]
```diff
@@ -0,0 +1,143 @@
...
+<br/>
+<a href="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step6.jpg" target="_blank">
+<img width="80%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step6.jpg" alt="">
+</a>
+<br/>
+
+<br/>
+ For My Parking Agent, I just told it to rephrase the way ask a question. Now, it will take in the feedback and rephrase the response accordingly.
+
+<br/>
```
Comment (amy17519) [inline content/en/seachat/getting-started/02-quick-start.md:128]: I don't really understand the value of this example. Maybe ask about a parking lot that does not exist in the dataset and then add it in the document? 

This example does not seem very critical to agent performance to me

File: content/en/seachat/getting-started/02-quick-start.md [hunk 18]
```diff
@@ -0,0 +1,143 @@
...
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step6-002.jpg" alt="">
+<br/>
+
+<br/>
+
+Fine-tuning your AI agent is a crucial and continuous process. We kindly advise you to check out the [**Advanced**]( /en/seachat/seachat-intro/03-add-knowledge/add-knowledge-intro) section in our user manual for a more comprehensive explanation and to understand the different parameters and features involved in the fine-tuning process.
```
Comment (amy17519) [inline content/en/seachat/getting-started/02-quick-start.md:134]: this "Advanced" hyperlink currently does not work

File: content/en/seachat/getting-started/01-seachat-intro.md [hunk 19]
```diff
@@ -0,0 +1,26 @@
+---
```
Comment (amy17519) [inline content/en/seachat/getting-started/01-seachat-intro.md:1]: - [ ] Cmd+F the entire document. Find and "appropriately" replace all `bot` to `agent`

File: content/en/seachat/getting-started/02-quick-start.md (hunk 17 above)
Comment (itIsJim) [reply content/en/seachat/getting-started/02-quick-start.md:128]: This makes a lot more sense. Will do.

**Review by amy17519 — APPROVED** [review]

**Review by xuchen — APPROVED** [review]
Great. I like the new feature that you can enlarge a picture.

Small requests (that need some work):
image

The picture sizes are still too big... We don't need 5Kx6K resolution, you can reduce the pic size by another 75% if limiting the resolution to lower such as 1024

**Comment by itIsJim** [comment]
> Great. I like the new feature that you can enlarge a picture.
> 
> Small requests (that need some work): image
> 
> The picture sizes are still too big... We don't need 5Kx6K resolution, you can reduce the pic size by another 75% if limiting the resolution to lower such as 1024

Got it. Will fix that.

**Pull request merged**